exercises/complex-numbers/test/test_complex_numbers.c

```diff
@@ -0,0 +1,365 @@
+#include "../src/complex_numbers.h"
+
+#include "vendor/unity.h"
+
+#include <math.h>
+
+#define PI acos(-1)
+#define E exp(1)
+
+void compare_complex(complex_t lhs, complex_t rhs)
+{
+   TEST_ASSERT_EQUAL(lhs.real, rhs.real);
```

Comment:
Use `TEST_ASSERT_EQUAL_DOUBLE` to compare doubles, this I believe boils down to just an '==' which shouldn't be used.